routes/tableRouter.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import { tableController } from '../controllers/tableController.js';
+
+export const tableRouter = express.Router();
+
+tableRouter
+  .get('/', tableController.getTables)
+  .post('/', tableController.addTable)
+  .get('/:id', tableController.getTable)
+  .put('/:id', tableController.updateTable)
+  .delete('/:id', tableController.deleteTable);
```

Comment:
add endpoint for getting all tables within one restaurant

Reply:
I don't know how to do this and I need help